.github/workflows/codingStandards.yml

```diff
@@ -11,8 +11,8 @@ jobs:
     name: PHPStan
     runs-on: ubuntu-latest
     steps:
-      - name: Set default PHP7.4
```

Comment:
Would it make sense to run the code against both PHP7.4 / PHP8.1?
Our code should be compatible with both versions (also according to our composer.json)

Reply:
@ronaldtebrake While it would make sense, but I am afraid of almost doubling tests because of two PHP versions.

Reply:
I have setup two tests for Github actions based on PHP 7.4 and PHP 8.0, Travis on PHP 8.0 as the new docker image would have PHP 8.0.15. See: https://github.com/goalgorilla/open_social_docker/pull/19 and https://github.com/goalgorilla/drupal_social/pull/365